Cargo.lock

```diff
@@ -455,7 +444,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "93f2635620bf0b9d4576eb7bb9a38a55df78bd1205d26fa994b25911a69f212f"
 dependencies = [
  "bitcoin_hashes",
- "rand_core 0.6.4",
+ "rand_core 0.5.1",
```

Comment:
> thanks, I needed to update rand_core for bip39 crate

It actually downgrades it, can you please explain why?

Reply:
I didn’t explicitly request a downgrade. It happened because Cargo resolves dependencies based on the semver-compatible version range specified (e.g., v1...v5), and it selects the highest compatible version available by default. But if another dependency restricts it to a lower version, Cargo might resolve to that instead.

I believe this change doesn't break, change anything we can leave it until we no longer rely on `rand_core 0.5.1` or we upgrade `bip39` crate

Reply:
> I believe this change doesn't break, change anything we can leave it until we no longer rely on rand_core 0.5.1 or we upgrade bip39 crate

We can also revert this to `rand_core 0.6.4` as I think it won't cause problems, will it? cargo.lock will be relying less on `rand_core 0.5.1` which makes it easier to remove it in the future. These left artifacts can persist over time as cargo might never resolve it on it's own if the dependency tree is complex.

Reply:
done